### PR TITLE
Pull Request for Issue1546: Added the transmission analysis block to the 2D scan on IDEAS.

### PR DESCRIPTION
**Pull request by dretrex**
Added the transmission analysis block to the 2D scan on IDEAS.

### CODE REVIEW

File: source/acquaman/IDEAS/IDEAS2DScanActionController.cpp [hunk 1]
```diff
@@ -93,7 +93,15 @@ IDEAS2DScanActionController::~IDEAS2DScanActionController()
 
 void IDEAS2DScanActionController::buildScanControllerImplementation()
 {
-	AMXRFDetector *detector = 0;
+    QList<AMDataSource *> i0Sources = QList<AMDataSource *>() << scan_->dataSourceAt(scan_->indexOfDataSource("I_0"));
+
+    AM2DNormalizationAB *transmissionAB = new AM2DNormalizationAB("transmission");
+    transmissionAB->setInputDataSources(i0Sources << scan_->rawDataSources()->at(scan_->rawDataSources()->indexOfKey("Sample")));
```
Comment (iainworkman) [inline source/acquaman/IDEAS/IDEAS2DScanActionController.cpp]: I'm assuming that all IDEAS2DScans have a raw datasource with the key "Sample"?

**Comment by iainworkman** [comment]
Other than the one comment this looks good.

**Comment by iainworkman** [comment]
Time Estimate: 10 Minutes

File: source/acquaman/IDEAS/IDEAS2DScanActionController.cpp (hunk 1 above)
Comment (dretrex) [inline source/acquaman/IDEAS/IDEAS2DScanActionController.cpp]: Yup.  Both are always added in the constructor.

**Pull request merged**